curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6553f8c570f9982e013a8886.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 6553f8c570f9982e013a8886
+title: Step 43
+challengeType: 20
+dashedName: step-43
+---
+
+# --description--
+
+Let's make the things more complex. Currently, each single letter is always encrypted with the same letter, depending on the specified offset. What if the offset is different for each letter? That would be much more difficult to decrypt.
+The Vigenère cipher can be considered the combination of many Caesar ciphers, where the offset for each letter is determined by another text, called key.
```

Comment:
```suggestion
The Vigenère cipher can be considered the combination of many Caesar ciphers, where the offset for each letter is determined by another text, called `key`.
```

Reply:
key needs to be translatable, so I've changed it into italic.